packages/contentful-source/gatsby-node.js

```diff
@@ -144,6 +144,19 @@ exports.createSchemaCustomization = ({ actions, schema }) => {
         infer: true,
       },
     }),
+    schema.buildObjectType({
+      name: 'ElviaFeature',
+      fields: {
+        name: 'String!',
+        isVisible: 'Boolean!',
+        id: 'ID!',
+        typeName: 'String',
```

Comment:
```suggestion
        typeName: 'String!',
```
Denne er vel "non-nullable" også?